_data/zarr.tsv

```diff
@@ -0,0 +1,47 @@
+Image	Name	Study	X	Y	Z	C	T	first_version
+179706	plate1_1_013 [Well 92, Field 1 (Spot 92)]	idr0002	1344	1024	1	2	329	0.1
+1884807	Centrin_PCNT_Cep215_20110506_Fri-1545_0_SIR_PRJ.dv	idr0021	256	256	1	3	1	0.1
+4007801	SPM00_TM000512_CM00_CM01_CHN01.klb	idr0044	2169	2048	988	2	532	0.1
+4495402	Sagittal	section	idr0053	921600	380928	1	1	0.1
```

Comment:
Something went wrong here as an unwanted tab was added, shifting the value.

Reply:
Fixed in https://github.com/IDR/idr.openmicroscopy.org/pull/108/commits/39f6425198116e66d58499112c39ae18a86456c3